coka/pagination-bundle/5.0/config/packages/oka_pagination.yaml

```diff
@@ -0,0 +1,4 @@
+oka_pagination:
+    db_driver: orm
```

Comment:
This looks like the same as the default config. 

If so, flex is smart enough to install this bundle without a recipe.

Reply:
Hi,

No this is not the default configuration. The `oka_pagination.db_driver` configuration must have a supplied value which must be either `orm` or `mongodb` but has no default value. If this value is not configured an exception should be thrown.

Reply:
Okey. Thank you. 

I think this is a good recipe then =)